frontend/document/koptinterface.lua

```diff
@@ -826,6 +826,9 @@ function KoptInterface:getTextFromBoxes(boxes, pos0, pos1)
                 line_text = line_text..word..space
             end
         end
+        -- append a space at the end of the line unless its a hyphenated word
+        line_text = line_text .. " "
+        line_text = line_text:gsub("- ", "")
```

Comment:
This would remove `"- "` in the middle of line_text. Should you match that only at end of line_text?

Reply:
You're quite right of course. :facepalm:  Not the biggest issue in most text I suppose but it should be `- $`.